cfgov/v1/handlers/blocks/feedback.py

```diff
@@ -14,7 +16,12 @@ def __init__(self, page, request, block_value):
 
     def process(self, is_submitted):
         if is_submitted:
-            form = FeedbackForm(self.request.POST)
+            if 'help-us-improve' in self.page.url:
```

Comment:
Is there some way to avoid having to hardcode these URL checks? Is using `self.block_value` an option?